cpp/example_code/s3/delete_bucket.cpp

```diff
@@ -51,8 +51,8 @@ bool AwsDoc::S3::DeleteBucket(const Aws::String& bucketName,
 int main()
 {
     //TODO: Change this bucket_name to the name of a bucket that actually exists in your account.

```

Comment:
Change: "Change this bucket_name to the name of a bucket that actually exists in your account." to "Change bucket_name to the name of a bucket in your account."